src/Extensions/Hosting.Services/Internal/ServiceFabricExecutionContext.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Fabric;
+using System.Net;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Omex.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services
+{
+	internal sealed class ServiceFabricExecutionContext : EmptyExecutionContext
+	{
+		public ServiceFabricExecutionContext(IHostEnvironment hostEnvironment, IServiceContextAccessor<ServiceContext> accessor)
+		{
+			MachineName = GetMachineName();
+			// In generic host context application is what it's running, Service Fabric service in this case, so it's application name is service fabric service name
+			ServiceName = hostEnvironment.ApplicationName;
+			EnvironmentName = hostEnvironment.EnvironmentName ?? DefaultEmptyValue;
+			IsPrivateDeployment = hostEnvironment.IsDevelopment();
+			RegionName = GetRegionName() ?? DefaultEmptyValue;
+
+			accessor.OnContextAvailable(UpdateState);
+		}
+
+
+		private string? GetRegionName() =>
+			Environment.GetEnvironmentVariable("REGION_NAME"); // should be defined by Azure http://whatazurewebsiteenvironmentvariablesareavailable.azurewebsites.net/
```

Comment:
I'd suggest changing the URL to the HTTPS equivalent. It stops unintelligent security scanners picking up "vulnerabilities" and it encourages users to follow best practices (by visiting the secure version of the website).

Reply:
I will resolve it in the follow-up pr https://github.com/microsoft/Omex/pull/168